### PR TITLE
Chap6

### PR DESCRIPTION
**Pull request by arxrobert**

### CODE REVIEW

**Comment by arxrobert** [comment]
Oops! Its Chap5 =)

File: app/assets/javascripts/users.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (Ekzosta) [inline app/assets/javascripts/users.coffee]: Remove extra file

File: app/assets/stylesheets/users.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Users controller here.
```
Comment (Ekzosta) [inline app/assets/stylesheets/users.scss]: Remove extra file

File: app/controllers/users_controller.rb [hunk 3]
```diff
@@ -0,0 +1,4 @@
+class UsersController < ApplicationController
+  def new
```
Comment (Ekzosta) [inline app/controllers/users_controller.rb]: Use syntax
def new; end

File: config/routes.rb [hunk 4]
```diff
@@ -1,7 +1,11 @@
 Rails.application.routes.draw do
   root 'static_pages#home'
-  get 'static_pages/home'
-  get 'static_pages/help'
-  get 'static_pages/about'
-  get 'static_pages/contact'
+
+  get '/help',    to: 'static_pages#help'
+
```
Comment (Ekzosta) [inline config/routes.rb]: Avoid extra line, if similar routes

File: config/routes.rb [hunk 5]
```diff
@@ -1,7 +1,11 @@
...
+  get '/about',   to: 'static_pages#about'
+
```
Comment (Ekzosta) [inline config/routes.rb]: Avoid extra line, if similar routes

**Comment by arxrobert** [comment]
Ready

**Pull request merged**